README.md

```diff
@@ -6,74 +6,52 @@
 
 Echidna is the central piece of software taking care of the new publication workflow at [W3C](http://www.w3.org/). The plan is for Echidna and related sub-projects (see below) to automatise the publication of new specs under [`http://www.w3.org/TR/`](http://www.w3.org/TR/).
 
-## Dependencies
+## Installation
 
-### OS-level dependencies
+To run Echidna, you need to install [Node.js](http://nodejs.org/) first.
+This will install [npm](https://www.npmjs.org/) at the same time, which is required as well.
 
-* [Node.js](http://nodejs.org/)
-* [npm](https://www.npmjs.org/)
-
-### npm dependencies
-
-These will be resolved automatically by simply running `npm install` on the directory of the project.
-
-* [express](https://github.com/strongloop/express)
-* [ejs](https://github.com/tj/ejs)
-* [Immutable.js](https://github.com/facebook/immutable-js/)
-* [Promise](https://github.com/then/promise)
-* [Moment](http://momentjs.com/)
-* [Specberus](https://github.com/w3c/specberus)
-* [`third-party-resources-checker`](https://github.com/dontcallmedom/third-party-resources-checker)
-
-## How to get it up and running
+Then run the following commands with your favorite terminal:
 
 ```bash
-$ git clone git@github.com:w3c/echidna.git
-$ cd echidna
-$ cp config.js.example config.js
-$ npm install
-$ npm start
+git clone https://github.com/w3c/echidna.git
+cd echidna
+cp config.js.example config.js
+npm install
```

Comment:
Removed the '$' is great. Makes it easy to copy/paste directly into console :thumbsup: